dotcom-rendering/src/lib/getZIndex.test.ts

```diff
@@ -26,7 +29,7 @@ describe('getZIndex', () => {
 		expect(getZIndex('searchHeaderLink')).toBe('z-index: 11;');
 		expect(getZIndex('TheGuardian')).toBe('z-index: 10;');
 		expect(getZIndex('editionSwitcherBanner')).toBe('z-index: 9;');
-		expect(getZIndex('headerWrapper')).toBe('z-index: 8;');
+		expect(getZIndex('tagLinkOverlay')).toBe('z-index: 8;');
```

Comment:
Oh yeah, looks like that's a mistake, it's not there on [main](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/getZIndex.ts#L72).

Happened when rebasing, the interface I got seemed to suggest this had been added recently rather than removed!